stock_card_report/reports/stock_card_report_xlsx.py

```diff
@@ -0,0 +1,137 @@
+# Copyright 2019 Ecosoft Co., Ltd. (http://ecosoft.co.th)
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+import logging
+from odoo import models
+
+_logger = logging.getLogger(__name__)
+
+
+class ReportStockCardReportXlsx(models.TransientModel):
+    _name = 'report.stock_card_report.report_stock_card_report_xlsx'
+    _inherit = 'report.report_xlsx.abstract'
+
+    def generate_xlsx_report(self, workbook, data, objects):
+        self._define_formats(workbook)
+        products = objects.results.mapped('product_id')
+        for product in products:
+            for ws_params in self._get_ws_params(workbook, data, product):
+                ws_name = ws_params.get('ws_name')
+                ws_name = self._check_ws_name(ws_name)
+                ws = workbook.add_worksheet(ws_name)
+                generate_ws_method = getattr(
+                    self, ws_params['generate_ws_method'])
+                generate_ws_method(
+                    workbook, ws, ws_params, data, objects, product)
+
+    def _get_ws_params(self, wb, data, product):
+
+        stock_card_template = {
+            '1_date': {
+                'header': {
+                    'value': 'Date',
+                },
+                'data': {
+                    'value': self._render('date'),
+                    'type': 'datetime',
+                    'format': self.format_tcell_date_left,
+                },
+                'width': 25,
+            },
+            '2_reference': {
+                'header': {
+                    'value': 'Reference',
+                },
+                'data': {
+                    'value': self._render('reference'),
+                },
+                'width': 25,
+            },
+            '3_input': {
+                'header': {
+                    'value': 'Input',
+                },
+                'data': {
+                    'value': self._render('input'),
+                    'format': self.format_tcell_amount_right,
+                },
+                'width': 20,
+            },
+            '4_output': {
+                'header': {
+                    'value': 'Output',
+                },
+                'data': {
+                    'value': self._render('output'),
+                    'format': self.format_tcell_amount_right,
+                },
+                'width': 20,
+            },
+            '5_balance': {
+                'header': {
+                    'value': 'Balance',
+                },
+                'data': {
+                    'value': self._render('balance'),
+                    'format': self.format_tcell_amount_right,
+                },
+                'width': 20,
+            },
+        }
+
+        ws_params = {
+            'ws_name': product.name,
+            'generate_ws_method': '_stock_card_report',
+            'title': 'Stock Card Report - '+product.name,
```

Comment:
```suggestion
            'title': 'Stock Card Report - %s' % product.name,
```

Reply:
I think I should to use this code `'title': 'Stock Card Report - {}'.format(product.name),`.